src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+>>>>>>> branch-Level-8
+
+
 public class Duke {
+    public static void printFileContents(String path) throws FileNotFoundException {
+        File f = new File(path); //create a file for the given file path
+        Scanner sc = new Scanner(f);
+        while (sc.hasNext()) {
+            System.out.println(sc.nextLine());
+        }
+    }
+
+    public static void writeToFile(String path, ArrayList<Task> list) throws IOException {
+        FileWriter fw = new FileWriter(path);
+        for (int i = 0; i < list.size(); i++) {
+            fw.write(list.get(i).formatText());
+        }
+        fw.close();
+    }
+
+    public static void appendToFile(String path, String text) throws IOException {
+        FileWriter fw = new FileWriter(path, true);
+        fw.write(text);
+        fw.close();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        System.out.println("Hello friend. What can I do for you? xD\n");
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> list = new ArrayList<Task>(100);
+        String path = "duke/src/main/java/duke.txt";
+
+        while (sc.hasNextLine()) {
+            try {
+                printFileContents(path);
+                String command = sc.nextLine();
+                String[] arr = command.split(" ", 2);
+
+                if (command.equals("bye")) {
+                    System.out.println("Bye!!! See ya again soon xD");
+                    break;
+                } else if (command.equals("list")) {
+                    System.out.println("Here are the tasks in your list:");
+                    for (int i = 0; i < list.size(); i++) {
+                        System.out.println((i + 1) + "." + list.get(i).toString());
+                    }
+                } else if (command.equals("find")) {
+
+                } else if (command.equals("todo") || command.equals("deadline") || command.equals("event")) {
+                    throw new DukeException("☹ OOPS!!! The description of a " + command + " cannot be empty.");
+
+                } else {
+                    Task t = new Task(command);
+                    if ((arr[0]).equals("done")) {
+                        int i = Integer.parseInt(arr[1]);
+                        list.get(i - 1).markDone();
+                        System.out.println("Nice! I've marked this as done:" + "\n  " + list.get(i - 1).toString());
+
+                    } else if ((arr[0]).equals("delete")) {
+                        int i = Integer.parseInt(arr[1]);
+                        Task d = list.get(i - 1);
+                        list.remove(d);
```

Comment:
Can implement SLAP here, by abstracting out the remove operation as well as printing out operation.